src/pykeen/models/unimodal/unstructured_model.py

```diff
@@ -51,40 +48,26 @@ def __init__(
         embedding_dim: int = 50,
         scoring_fct_norm: int = 1,
         loss: Optional[Loss] = None,
+        predict_with_sigmoid: bool = False,
         preferred_device: DeviceHint = None,
         random_seed: Optional[int] = None,
-        regularizer: Optional[Regularizer] = None,
         entity_initializer: Hint[Initializer] = xavier_normal_,
     ) -> None:
         r"""Initialize UM.
 
         :param embedding_dim: The entity embedding dimension $d$. Is usually $d \in [50, 300]$.
         :param scoring_fct_norm: The $l_p$ norm. Usually 1 for UM.
         """
+        self.embedding_dim = embedding_dim
```

Comment:
is this necessary?

Reply:
I guessed it was for testing purposes, but I managed to remove it and nothing started failing